### DISCUSSION TITLE
Why is my highlight group not overwriting?

### DISCUSSION THREAD

**Discussion by chinleung** [nvim-treesitter/nvim-treesitter#6924] [Q&A]
Hi,

I have the following PHP code:

```php
return AccountUser::where(/* ... */)->get();
```

And the `::where` is highlighted in green:

<img width="409" alt="image" src="https://github.com/user-attachments/assets/e2c1d2d6-d9e5-41b0-bfcc-acef98d324c3">

I'm trying to change it to `GruvboxWhite` by adding this to my `highlights.scm`:

```scm
(scoped_call_expression
  name: (name) @function.method.call)
```

And this to my `.lua` file:

```lua
vim.cmd('hi link @function.method.call.php GruvboxWhite')
```

If I run the command `:Inspect`, I see that my `@function.method.call.php` has been added properly to the `where` since I get this as the result:

```
Treesitter
  @function.call.php links to GruvboxGreenBold php
  @function.method.call.php links to GruvboxWhite php
```

But my colour is still set to `GruvboxGreenBold`. 🧐 

Can anyone guide me on what I'm doing wrong please? 🙏🏼 

Thanks!

**Comment by clason** [comment]
https://gist.github.com/romainl/379904f91fa40533175dfaec4c833f2f
